packages/react-ai/src/components/AIConversation/views/default/MessageList.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { Avatar, Text, View } from '@aws-amplify/ui-react';
+import { MessageControl } from '../Controls/MessagesControl';
+import {
+  AvatarsContext,
+  MessageVariantContext,
+  RoleContext,
+} from '../../context';
+import { ConversationMessage } from '../../../../types';
+import { formatDate } from '../../utils';
+import { ControlsContextProps } from '../../context/ControlsContext';
+import {
+  ComponentClassName,
+  classNameModifier,
+  classNames,
+} from '@aws-amplify/ui';
+
+const MessageMeta = ({ message }: { message: ConversationMessage }) => {
+  // need to pass this in as props in order for it to be overridable
+  const avatars = React.useContext(AvatarsContext);
+  const role = React.useContext(RoleContext);
+  // maybe rename 'avatar' to something else
+  const avatar = role === 'assistant' ? avatars?.ai : avatars?.user;
+  return (
+    <View className={ComponentClassName.AIConversationMessageSender}>
+      <Text className={ComponentClassName.AIConversationMessageSenderUsername}>
+        {avatar?.username}
+      </Text>
+      <Text className={ComponentClassName.AIConversationMessageSenderTimestamp}>
```

Comment:
We shouldn't be using `ComponentClasses` to avoid bundle size impacts

Reply:
These are the default amplify components which use our primitives can discuss on monday